GitUpKit/Views/GIMapViewController+Operations.m

```diff
@@ -395,6 +395,12 @@ - (void)editCommitMessage:(GCHistoryCommit*)commit {
                           }];
 }
 
+- (void)copyCommitMessage:(GCHistoryCommit*)commit {
+  NSString* message = _CleanedUpCommitMessage(commit.message);
+  [self.windowController showOverlayWithStyle:kGIOverlayStyle_Informational format:NSLocalizedString(@"Commit message copied: \"%@\"", nil), message];
+  [[NSPasteboard generalPasteboard] setString:message forType:NSPasteboardTypeString];
```

Comment:
You don't have any `-[NSPasteboard declareTypes:owner:]` call. It's unlikely to be copying anything to the pasteboard in this case. From the docs: "The type must have been declared by a previous declareTypes:owner: message."

Reply:
@lucasderraugh Heh, my bad. It is ridiculous how different macOS and iOS at some points. Yes, all applications in iOS are sandboxed, so, they can't harm user.

Mmm, I've found a bug in current release app. I think that "testing" code by others is one of the best practices to find bugs.

In this PR I was in hurry, yes. It is one of the core functions that I need ( besides drag & drop, history of file, search in commit view  and search in selected range of commits ).